tests/chainer_tests/functions_tests/connection_tests/test_convolution_nd.py

```diff
@@ -82,7 +91,15 @@ def setUp(self):
                 'atol': 2 ** -4, 'rtol': 2 ** -4})
 
     def before_test(self, test_name):
-        self.backend_config.autotune = self.autotune
```

Comment:
I set the autotune param here to the backend_config

Reply:
This causes several tests to fail with `u'CUDNN_STATUS_NOT_SUPPORTED`
This change must be reverted